### ISSUE TITLE
About pretrained_model

### ISSUE THREAD

**Issue by XiaoYangon** [geekyutao/RN#6]
Hello, what is the train data of pretrained model?

**Comment by geekyutao** [comment]
Hi, it's Places2 training dataset with irregular mask dataset[1].

[1] [Image Inpainting for Irregular Holes Using Partial Convolutions](https://nv-adlr.github.io/publication/partialconv-inpainting)
